resources/viewer-js-hash

```diff
@@ -0,0 +1 @@
+yh4majqZ75XpcUpjEFqRovMNjpk
```

Comment:
Maybe a simplification / generalization would be to save the full lookup url here?

Reply:
That would not be a simplication because in some cases we need the gs:// url and in other cases we need the http:// url.